configuration/restdocs.md

```diff
@@ -7,14 +7,29 @@ title: openHAB REST API
 
 # openHAB REST API
 
-The REST API of openHAB serves different purposes.
-It can be used to integrate openHAB with other systems as it allows read access to items and item states as well as status updates or the sending of commands for items.
-Furthermore, it gives access to the sitemaps, so that it is the interface to be used by remote user interfaces (e.g. fat clients or javascript based web clients).
-The REST API also supports server-push, so you can subscribe yourself on change notification for certain resources.
```

Comment:
The last line here describes an important feature and should not be deleted, in fact I'd give it its own headline.

Reply:
I agree - and I included it in the version I posted in the discussion we had been having.

Reply:
Thanks added it back in.

Reply:
Don't know how to highlight specific sections from below.  So here we go.  Below, recommend changing: are based on a simple http protocoll.

to this:
are based on http protocol.

Reply:
Ahh - now I found it - on my mac, press the blue '+' sign.  Geez - so much to learn.